lib/snippet-expansion.coffee

```diff
@@ -10,17 +10,37 @@ class SnippetExpansion
     @selections = [@cursor.selection]
 
     startPosition = @cursor.selection.getBufferRange().start
+    body = @snippet.body
+    indent = ""
+    tabStopRanges = []
+    if @snippet.lineCount > 1
+      lines = body.split('\n')
+      indent = @editor.lineTextForBufferRow(startPosition.row).match(/^\s*/)[0]
+      for line, index in lines when index isnt 0 # Do not include initial line
+        # Match initial line's indent
+        lines[index] = indent + line
+      body = lines.join('\n')
```

Comment:
`body = @snippet.body.replace('\n', '\n' + indent)`

Reply:
Can you explain this change?  I'm not sure I follow.

Reply:
You could replace the code above with
```coffee
indent = @editor.lineTextForBufferRow(startPosition.row).match(/^\s*/)[0]
body = @snippet.body.replace('\n', '\n' + indent)
tabStopRanges = []
```